src/redux/reducers/tripReducers.js

```diff
@@ -0,0 +1,141 @@
+import {
+  GET_TRIPS_REQUEST,
+  GET_TRIPS_FAILURE,
+  GET_TRIPS_SUCCESS,
+  ADD_TRIP_REQUEST,
+  ADD_TRIP_FAILURE,
+  ADD_TRIP_SUCCESS,
+  UPDATE_TRIP_REQUEST,
+  UPDATE_TRIP_FAILURE,
+  UPDATE_TRIP_SUCCESS,
+  DELETE_TRIP_REQUEST,
+  DELETE_TRIP_FAILURE,
+  DELETE_TRIP_SUCCESS,
+  GET_TRIP_INFO_REQUEST,
+  GET_TRIP_INFO_FAILURE,
+  GET_TRIP_INFO_SUCCESS
+} from '../actions/tripActions';
```

Comment:
I feel like this should be done the same way as in rootReducer. ie 
`import * as TripActions from '../actions/tripActions';`

Reply:
I agree. Will change this now.